src/interface/src/app/plan/plan-map/opacity-control/opacity-control.component.scss

```diff
@@ -0,0 +1,27 @@
+.panel-container {
+    height: 34px;
+    padding: 0;
+}
+
+.opacity-container {
+    display: contents;
```

Comment:
unfortunately, this display value is not widely supported yet  :( 
https://caniuse.com/css-display-contents

Reply:
Removed, thank you!